File: BiosenseSignal_Flutter_SDK_5.6.3/ios/Classes/NativeBridgeEvents.swift
//
//  NativeBridgeEvents.swift
//  biosensesignal_flutter_sdk
//
//  Created on 26/03/2023.
//

import Foundation

class NativeBridgeEvents {
    static let imageData = "image_data"
    static let sessionStateChange = "session_state_change"
    static let sessionWarning = "session_warning"
    static let sessionError = "session_error"
    static let sessionVitalSign = "session_vital_sign"
    static let sessionFinalResults = "session_final_results"
    static let enabledVitalSigns = "enabled_vital_signs"
    static let licenseInfo = "license_info"
    static let ppgDeviceDiscovered = "ppg_device_discovered"
    static let ppgDeviceScanFinished = "ppg_device_scan_finished"
    static let ppgDeviceInfo = "ppg_device_info"
    static let ppgDeviceBatteryLevel = "ppg_device_battery_level"
    static let fallDetectionData = "fall_detection_data"
}

File: BiosenseSignal_Flutter_SDK_5.6.3/android/src/main/kotlin/com/biosensesignal/flutter_sdk/NativeBridgeEvents.kt
package com.biosensesignal.flutter_sdk

class NativeBridgeEvents {

    companion object {
        const val imageData = "image_data"
        const val sessionStateChange = "session_state_change"
        const val sessionWarning = "session_warning"
        const val sessionError = "session_error"
        const val sessionVitalSign = "session_vital_sign"
        const val sessionFinalResults = "session_final_results"
        const val enabledVitalSigns = "enabled_vital_signs"
        const val licenseInfo = "license_info"
        const val ppgDeviceDiscovered = "ppg_device_discovered"
        const val ppgDeviceScanFinished = "ppg_device_scan_finished"
        const val ppgDeviceInfo = "ppg_device_info"
        const val ppgDeviceBattery = "ppg_device_battery_level"
        const val fallDetectionData = "fall_detection_data"
    }
}